packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -66,6 +67,38 @@ export class RemoteMediaGroup {
     return [...this.unpinnedRemoteMedia, ...this.pinnedRemoteMedia];
   }
 
+  /**
+   * fixCsis
+   *
+   */
+  public fixCsis(
+    remoteMediaCsis: {remoteMedia: RemoteMedia; csi?: number | undefined}[],
+    commit = true
+  ): void {
+    forEach(remoteMediaCsis, ({csi, remoteMedia}) => {
+      if (csi) {
+        if (!(this.pinnedRemoteMedia.indexOf(remoteMedia) >= 0)) {
+          const unpinId = this.unpinnedRemoteMedia.indexOf(remoteMedia);
```

Comment:
we need to handle a case where the remoteMedia is not in either pinned or unpinned list - basically it's not in the group